pttrack/followup_models.py

```diff
@@ -141,9 +143,11 @@ def short_text(self):
 class ReferralFollowup(Followup):
     '''Datamodel for a PCP referral followup.'''
 
+    # Template relies on following variable to render Admin Edit. If you change the variable here, you must edit patient_detail.html
     REFTYPE_HELP = "What kind of provider was the patient referred to?"
     referral_type = models.ForeignKey(mymodels.ReferralType,
-                                      help_text=REFTYPE_HELP)
+                                      help_text=REFTYPE_HELP, 
+                                      default="Other")
```

Comment:
Why are we not requiring this? @radhikajain 

Referral to the COH can mean a lot of things, for example...